src/groovy/org/pih/warehouse/tablero/MultipleNumbersIndicator.groovy

```diff
@@ -0,0 +1,20 @@
+package org.pih.warehouse.tablero
+
+import org.pih.warehouse.tablero.ColorNumber
+
+class MultipleNumbersIndicator implements Serializable {
+    
+    List<ColorNumber> listColorNumber;
+
+    MultipleNumbersIndicator(List<ColorNumber> listColorNumber) {
+        this.listColorNumber = listColorNumber;
+    }
+
+    Map toJson() {
+        for(int i=0;i<listColorNumber.size();i++)
```

Comment:
Could you add curly brackets and return for better readability?

Reply:
Also please add spaces around operators and after 'for'.

Reply:
for (int i = 0; i < listColorNumber.size(); i++) {

Reply:
I just changed in : 

for (int i=0;i<listColorNumber.size();i++) {
            listColorNumber[i] = listColorNumber[i].toJson();
        }

Reply:
I just changed it